lucien-ios/LucienConstants.swift

```diff
@@ -43,3 +60,4 @@ final class LucienConstants {
     static let overlayButtonTopConstraint = CGFloat(10)
     static let overlayButtonLeadingConstraint = CGFloat(15)
 }
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Newline Violation: Files should have a single trailing newline. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_newline)